packages/core/docs/advanced/calling-platform-api.md

```diff
@@ -0,0 +1,25 @@
+# Calling integration APIs
+
+## How do Integrations work?
+
+In Vue Storefront, single integration has is own sort of SDK library (named API-client), that contains set of functions. Each function is dedicated to one action, endpoint or a feature eg. `getProduct`, `loadCart` `addToCart`. Therefore, when you call for instance, `getProduct`, it will request the integration API and return its response in the original format of given platform.
+
+## Accessing integration methods on the frontend
+
+To access API-client functions, you can use the composable function `useVSFContext` that reads from the application context every possible integration so you can easily access it.
```

Comment:
**suggestion**: I believe the function name is explanatory here and we can shorten this sentence:

```suggestion
To access API-client functions, you can use the composable function `useVSFContext`:
```